src/components/layout/table-row/Row.jsx

```diff
@@ -155,11 +156,16 @@ export class Row extends Component {
 
         addEmptyInsert(cells, visibleColumns, plugins);
 
-        const rowEl = (
-            <tr { ...rowProps }>
-                { cells }
-            </tr>
-            );
+        let rowEl;
+        if(rowRenderer && typeof rowRenderer  === 'function'){
+          rowEl = rowRenderer({row, rowProps, cells});
+        }else{
+          rowEl = (
+              <tr { ...rowProps }>
+                  { cells }
+              </tr>
+              );
+        }
```

Comment:
With this implementation, we're ignoring the drag and drop if a custom renderer is passed. We probably still want the drag and drop to work, so I would work the logic a bit differently here,